File: scripts/agent-bench/fixture-php/src/DocEntry.php
<?php

declare(strict_types=1);

namespace Corpus;

class DocEntry
{
    public function __construct(
        public string $name,
        public string $path,
        public float $score,
        public EntryKind $kind,
    ) {
    }

    public function display(): string
    {
        return "{$this->name} ({$this->path})";
    }
}

File: scripts/agent-bench/fixture-java/src/main/java/com/corpus/DocEntry.java
package com.corpus;

public class DocEntry {
    public String name;
    public String path;
    public float score;
    public EntryKind kind;

    public DocEntry(String name, String path, float score, EntryKind kind) {
        this.name = name;
        this.path = path;
        this.score = score;
        this.kind = kind;
    }

    public String display() {
        return name + " (" + path + ")";
    }
}
